packages/@icon-magic/cli/src/index.ts

```diff
@@ -54,46 +59,70 @@ program
   });
 
 program
-  .command(
-    'distribute',
+  .command('distribute [inputPaths...]')
+  .description(
     'moves an icon from the source to the destination, applying types as specified'
   )
-  .option('-i, --inputPaths', 'path to the input directory of icons')
   .option(
-    '-o, --outputPath',
+    '-o, --outputPath [outputPath]',
     'path to the output directory where the generated assets are to be written to'
   )
   .option(
-    '-t, --type',
+    '-t, --type [type]',
     'type of icons format to handle, accepted types are svg|png|webp'
   )
   .option(
-    '-g, --groupByCategory',
+    '-g, --groupBy [groupBy]',
     '[for web sprite creation] if to group the icons by category'
   )
-  .action(async (i: string, o: string, t: ICON_TYPES, g: boolean) => {
-    if (!i.length) {
+  .action(async (inputPaths, options) => {
+    if (!inputPaths.length) {
       LOGGER.error('No Input Directories were specified.\n');
+      process.exit(1);
+    }
+
+    // throw an errof if outputPath isn't specified
+    if (!options.outputPath) {
```

Comment:
Add as part of synopsis (instead of an option) if required?

Reply:
Want to support the use case of having multiple inputPaths.